### PR TITLE
Expose 'group' and 'project' to insert_run_start

### LINKED ISSUE #91: Project field in insert_run_start
The `project` field in the RunStart object needs to be exposed through the `insert_run_start` function

### PR DESCRIPTION
**Pull request by danielballan**
Closes #91 and #112. Merge after #118 -- this builds on commits in #118.

### CODE REVIEW

File: metadatastore/commands.py [hunk 1]
```diff
@@ -117,18 +117,24 @@ def insert_run_start(time, beamline_id, beamline_config=None, owner=None,
...
         The date/time as found at the client side when an event is
         created.
-    beamline_id: str
+    scan_id : int
+        Unique scan identifier visible to the user and data analysis
+    beamline_id : str
         Beamline String identifier. Not unique, just an indicator of
         beamline code for multiple beamline systems
-    beamline_config: metadatastore.odm_temples.BeamlineConfig, optional
+    beamline_config : metadatastore.odm_temples.BeamlineConfig
```
Comment (ericdill) [inline metadatastore/commands.py:125]: ~~maybe make this object `metadatastore.api.BeamlineConfig`~~

nevermind

**Comment by tacaswell** [comment]
Can you push a dummy commit to this and then forcepush it out of existence?  I think gh needs to be kicked to re do the diffs.

**Comment by tacaswell** [comment]
and does this or #118 have companion PRs into ophyd?

**Comment by danielballan** [comment]
@tacaswell Github has been kicked. This does not have a companion PR in ophyd. But I believe @dchabot has some commits making it possible to pass arguments through `*scan` to `insert_run_start`, making all of these fields exposed, if not documented.

**Comment by tacaswell** [comment]
I was more worried about the fact that the order of the args changed.

**Comment by danielballan** [comment]
This is why we need Travis set up for ophyd. I think we were careful to use
kwargs but we should check (tomorrow) before we merge.

**Comment by danielballan** [comment]
Confirmed: the only call to `insert_run_start` in ophyd passes everything by keyword argument. This change is safe.

**Comment by danielballan** [comment]
And likewise with the other `insert_*` calls, whose `uid` kwarg has been reordered by this PR.

**Pull request merged**